gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Thu Jun 30 15:08:56 CEST 2016
```

Comment:
Actually I just read how the travis works, and as far as I got it, it builds with the wrapper if it's comitted. So i think it's fine.